app/controllers/evidence_items_controller.rb

```diff
@@ -34,6 +34,21 @@ def variant_index
     )
   end
 
+  def variant_group_index
+    variants = EvidenceItem.variant_group_scope
+      .order('evidence_items.id asc')
+      .page(params[:page].to_i)
+      .per(params[:count].to_i)
+      .where(variant_groups: { id: params[:variant_id] })
```

Comment:
Good catch. I basically copied this method from the variants controller, which also contains this bug. I made a PR to fix the variants controller (#356).